### PR TITLE
Prepare for 0.8 release

### PR DESCRIPTION
**Pull request by k0da**
Signed-off-by: Dinar Valeev <dinar.valeev@absa.africa>

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -112,8 +112,8 @@ deploy-full-local-setup: ## Deploy full local multicluster setup (k3d >= 4.2.0)
 .PHONY: deploy-stable
 deploy-stable:
 	@echo "\n$(YELLOW) import $(CYAN)k8gb:$(VERSION) $(YELLOW)to $(CYAN)$(CLUSTER_GSLB1), $(CLUSTER_GSLB2) $(NC)"
-	$(call deploy-local-cluster,$(CLUSTER_GSLB1),$(CLUSTER_GSLB2),$(VERSION),,'k8gb/k8gb')
-	$(call deploy-local-cluster,$(CLUSTER_GSLB2),$(CLUSTER_GSLB1),$(VERSION),$(CLUSTER_GSLB2_HELM_ARGS),'k8gb/k8gb')
+	$(call deploy-local-cluster,$(CLUSTER_GSLB1),$(CLUSTER_GSLB2),,,'k8gb/k8gb')
+	$(call deploy-local-cluster,$(CLUSTER_GSLB2),$(CLUSTER_GSLB1),,$(CLUSTER_GSLB2_HELM_ARGS),'k8gb/k8gb')
```
Comment (ytsarev) [inline Makefile:116]: https://github.com/AbsaOSS/k8gb/blob/bf5bc287c207326cc70bd9ab164f3ed34176f8c8/Makefile#L353
will be confusingly empty
Consider changing
```
@echo "\n$(YELLOW)Deploy GSLB operator from $3 $(NC)"
```
to
```
@echo "\n$(YELLOW)Deploy GSLB operator from ${3:-stable chart} $(NC)"
```

**Review by ytsarev — CHANGES_REQUESTED** [review]

File: Makefile (hunk 1 above)
Comment (k0da) [reply Makefile:116]: @ytsarev thanks for suggestion, amended

Comment (ytsarev) [reply Makefile:116]: https://github.com/AbsaOSS/k8gb/pull/471/checks?check_run_id=2534570627#step:5:17 unfortunately it takes empty string as a value

Comment (k0da) [reply Makefile:116]: @ytsarev which is fine, it takes "released" image then.

Comment (ytsarev) [reply Makefile:116]: @k0da i'm talking about empty value in the log statement

Comment (k0da) [reply Makefile:116]: @ytsarev is it confusing? I think making it conditional will complicates make file quite a bit. I don't see harm setting a value to be empty.

Comment (ytsarev) [reply Makefile:116]: @k0da 
image
Yes, `Deploy GSLB operator from <nothing>` is a confusing incomplete statement.

File: Makefile [hunk 2]
```diff
@@ -350,8 +350,8 @@ define deploy-local-cluster
 	@echo "\n$(YELLOW)Create namespace $(NC)"
 	kubectl apply -f deploy/namespace.yaml
 
-	@echo "\n$(YELLOW)Deploy GSLB operator from $3 $(NC)"
-	$(call deploy-k8gb-with-helm,$1,$2,$3,$4,$5)
+	@echo "\n$(YELLOW)Deploy GSLB operator from ${3} $(NC)"
+	$(call deploy-k8gb-with-helm,$1,$2,${3:"stable"=""},$4,$5)
```
Comment (ytsarev) [inline Makefile:354]: If we always pass `STABLE_VERSION`, why do we need `"stable"=""` ?

Comment (ytsarev) [reply Makefile:354]: fancy, got it :+1:

**Pull request merged**